hr_employee_gravatar/models/res_users.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2015 Endika Iglesias
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.one
+    def get_gravatar_image(self):
+        super(ResUsers, self).get_gravatar_image()
+        hr_employee = self.env['hr.employee'].search(
+            [('user_id', '=', self.id)])
```

Comment:
@Endika it's the same thing as your search. The problem is that your search expression will likely take twice the time because of the domain parsing overhead.